arlas-tests/src/test/java/io/arlas/server/tests/rest/collections/CollectionServiceIT.java

```diff
@@ -100,14 +101,18 @@ public void test02GetAllCollections() throws Exception {
         Map<String, Object> jsonAsMap = getJsonAsMap();
         jsonAsMap.put(CollectionReference.INSPIRE_PATH, getInspireJsonAsMap());
         jsonAsMap.put(CollectionReference.DUBLIN_CORE_PATH, getDublinJsonAsMap());
+        jsonAsMap.put(CollectionReference.ORGANISATIONS, getOrganisationsJsonAsMap(false));
 
         // PUT new collection 1
         given().contentType("application/json").body(jsonAsMap)
                 .when().put(arlasPath + "collections/collection1")
                 .then().statusCode(200);
 
         // GET all collections
-        getAllCollections(COLLECTION_NAME, COLLECTION_NAME_ACTOR, "collection1");
+//        getAllCollections(COLLECTION_NAME, COLLECTION_NAME_ACTOR, "collection1");
+//        getAllCollectionsWithOrg(DataSetTool.DATASET_ORG_OWNER, COLLECTION_NAME, COLLECTION_NAME_ACTOR, "collection1");
```

Comment:
is there a reason why these tests are commented ?

Reply:
The line comment was a mistake